qiling/os/posix/syscall/poll.py

```diff
@@ -4,6 +4,34 @@
 #
 
 from qiling import Qiling
+import select
 
 def ql_syscall_poll(ql: Qiling, fds: int, nfds: int, timeout: int):
-    return 0
\ No newline at end of file
+    fn_map = {}
+    try:
+        p = select.poll()
```

Comment:
This is going to break on a Windows host, as it only supports `select`.
Perhaps we need to have two implementations, based on the hosting system.

Reply:
As i know, `select()` canot handle events like `poll()`, so i just limit the syscall implemention in Linux host. Do you have a better solution?

Reply:
No, that is exactly what I meant.